cd/mxnet_lib/dynamic/Jenkins_pipeline.groovy

```diff
@@ -24,7 +24,7 @@
 
 // NOTE: the following variables are referenced in the mxnet_lib_pipeline jenkins file imported bellow
 // libmxnet location
-libmxnet = 'lib/libmxnet.so'
```

Comment:
As with the 1.7 PR: Changing this line is done when switching to cmake build on CD. I don't think it's done yet (but we certainly want to do it at some point to decrease the infrastructure code divergence between 1.x and master branch)

Reply:
I reverted this line. Rest of the code changes look good?